packages/math-inline/configure/src/response.jsx

```diff
@@ -130,7 +132,23 @@ class Response extends React.Component {
     onResponseChange(newResponse, index);
   };
 
-  onAnswerChange = answer => {
+  onAllowTrailingZerosChange = (event) => {
+    const { response, onResponseChange, index } = this.props;
+    const newResponse = { ...response };
+
+    newResponse.allowTrailingZeros = !response.allowTrailingZeros;
```

Comment:
Maybe we can create a custom function that gets a key and we'll have something like 
newResponse[key]= !response[key];

Reply:
done